tools/depends/target/flatbuffers/Makefile

```diff
@@ -49,6 +49,9 @@ ifeq ($(PREFIX),)
 endif
 	rm -rf $(PLATFORM)/*; mkdir -p $(PLATFORM)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
+ifeq ($(BUILD_FLATC),ON)
+	cd $(PLATFORM); patch -p1 < ../../../native/flatbuffers-native/0001-Fix-compiler-warning.patch
```

Comment:
@wsnipex okay to use the native patch or should it be copied into target?

Reply:
yes, no need to copy it